app/app.js

```diff
@@ -34,7 +34,7 @@ openSansObserver.check().then(() => {
 });
 
 // Import the CSS reset, which HtmlWebpackPlugin transfers to the build folder
-import '../node_modules/sanitize.css/dist/sanitize.min.css';
+import '../node_modules/sanitize.css/sanitize.css';
```

Comment:
To compile from source? :) Sanitize.css big +1 over normalize in my opinion is the fact it's highly configurable with css vars (or whatever).

However - this setup will require us to change the webpack settings. After I made that change I've noticed lots of broken CSS in my code. It turned out I had to enable postcss loader for node_modules as well, so it can parse css vars and stuff. I am also using it to compile basscss from source with custom settings.